src/FirstSteps/RPG/Adventures/Inventory.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FirstSteps.RPG.Heroes;
+using FirstSteps.RPG.Items;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FirstSteps.RPG.Adventures
+{
+    public class Inventory
+    {
+        private static List<Item> _inventory = new List<Item>()
+        {
+            new MagicAxe(),
+            new MagicBow(),
+            new MagicSkull(),
+            new MagicSword()
+        };
+        public void Inventoryy(Hero _hero)
```

Comment:
You have a typo in the method name `Inventoryy` - a double yy. On the other hand, it's quite a bad name because right now your class is named `Inventory`. Maybe it would be better to name the method in a way that describes an action that the hero does with the Inventory.